docs/graphkb/loading_data.md

```diff
@@ -40,6 +50,47 @@ Most popular resources which have pre-built loaders provided for GraphKB are lis
 
     Drug definitions and relationships can be loaded from ChEMBL via their REST API.
 
+- ![logo](https://docs.civicdb.org/en/latest/_images/CIViC_logo_for-light-bg_SM_v5a1.png)
+
+    ### CIViC
+
+    [https://civicdb.org](https://civicdb.org/)
+
+    [CC0 1.0](https://docs.civicdb.org/en/latest/about/faq.html?highlight=license#how-is-civic-licensed)
+
+    This is an external knowledge base which can be imported as statements into GraphKB
```

Comment:
here are in the similar statement above - end this with a period (for consistency with other statements) and perhaps use knowledgebase instead of knowledge base (again, for consistency); i think you've gone with knowledgebase elsewhere

Reply:
I've tried to go with "knowledge base" everywhere unless it is an external tool name where they call it "knowledgebase" but I might have missed some.

Going point on the periods. I'll try to go through and add them. Probably easier to just add them than remove them all

Reply:
I fixed the couple instances I missed, they should all be "knowledge base" now